### ISSUE TITLE
No Group Capability?

### ISSUE THREAD

**Issue by PrasannaVenkadesh** [glauth/glauth#391]
Hi. I am pretty new to LDAP stuff. So enlighten me if I am wrong.

Users can belong to groups. Is it not possible to add capabilities for groups instead of adding to each and every individual user?

**Comment by Fusion** [comment]
No gonna lie: my ultimate goal was to offer inherited capabilities so yes, this should exist. But it doesn't, and I worked on other things since then.
If you feel it would be a helpful improvement, I'll add it to the roadmap for this quarter.

**Comment by PrasannaVenkadesh** [comment]
Yes. I was evaluating if I can use glauth for my organization and then I saw this limitation.

**Comment by eubyte** [comment]
Having tested the 'groups.capabilities' feature (for search), it does not seem to work even though it is mentioned in the [sample configuration file](https://github.com/glauth/glauth/blob/master/v2/sample-simple.cfg). 1+ for implementing this very important feature. Thank you!
